playbooks/08-run-tests.yml

```diff
@@ -9,18 +9,10 @@
   gather_facts: false
   tasks:
     - name: "Run tests"
-      vars:
-        tempest_bool: cifmw_run_tempest | default('true') | bool
-        tobiko_bool: cifmw_run_tobiko | default('false') | bool
-        test_role: >-
-          {{ cifmw_run_test_role |
-             default('tempest' if tempest_bool
-                     else ('tobiko' if tobiko_bool else None))
-          }}
       tags:
         - tests
       ansible.builtin.import_role:
-        name: "{{ test_role }}"
+        name: "{{ cifmw_run_test_role | default('tempest') }}"
```

Comment:
is this a new top-level parameter? If so, care to ensure it's mentioned in this doc:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/usage/01_usage.md#top-level-parameters

Reply:
I have added a new commit documenting this and some other top level parameters.